go/pserver/optimizer.go

```diff
@@ -42,12 +42,12 @@ func newOptimizer(paramWithConfigs ParameterWithConfig) *optimizer {
 	c := paramWithConfigs.Config
 	log.WithFields(log.Fields{
 		"ElementType": p.ElementType,
-		"ParamSize":   len(p.Content),
+		"ParamSize":   len(p.Content) / C.sizeof_float,
 		"ConfigSize":  len(c),
 	}).Info("New Optimizer Created with config:")
 	var cbuffer unsafe.Pointer
 	cbuffer = C.malloc(C.size_t(len(p.Content)))
-	C.memcpy(cbuffer, unsafe.Pointer(&p.Content[0]), C.size_t(len(p.Content)))
+	C.memcpy(cbuffer, unsafe.Pointer(&p.Content[0]), C.size_t(len(p.Content)/C.sizeof_float))
```

Comment:
it is so strange that we can pass the `go test` smoothly.
what's more, `C.memcpy` copy four times longer buffer than the real length, it never core dump(system call libc's memcpy in low level), and set all the data to 0 as normal.